app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.length)
+  end
```

Comment:
- [ ] Nice use of `after_save` and making the `update_comments_counter` private. Keep rocking 🔥